rkt/run.go

```diff
@@ -256,6 +259,18 @@ func runRun(args []string) (exit int) {
 		return 1
 	}
 
+	if flagInteractive {
+		ac, err := p.getAppCount()
+		if err != nil {
+			stderr("run: cannot get container's app count: %v", err)
+			return 1
+		}
+		if ac > 1 {
+			stderr("run: interactive option only supports containers with one app")
+			return 1
+		}
+	}
```

Comment:
@philips this isn't a bugfix since we already have the check on line 204 (which is slightly redundant with this, but probably worth keeping for now as an optimisation)